recipe/meta.yaml

```diff
@@ -64,7 +65,7 @@ requirements:
   run:
     - llvm-spirv-{{ llvm_version.split(".")[0] }}
     - ocl-icd                            # [linux]
-    - khronos-opencl-icd-loader          # [osx]
+    - khronos-opencl-icd-loader <2024.10        # [osx]
```

Comment:
This should not be necessary right?

Reply:
I think you're right, removed the restriction in [39a6248](https://github.com/conda-forge/pocl-feedstock/pull/106/commits/39a6248f18bac5178764b946487479aa9b05fa54)